indispensable-asn1/build.gradle.kts

```diff
@@ -65,6 +65,7 @@ kotlin {
             )
 
             dependencies {
+                implementation(project(":internals"))
```

Comment:
I don't think having that in internals will work, since we want to publicly expose it. i think the indispensable module is the right place for that interface tandem, since we need it in all modules that depend on indispensable (wich are all others,  and we don't need it in indispensale-asn1, wich is the only one that does not depend on it)